code/game/objects/items/stacks/medical.dm

```diff
@@ -39,7 +39,7 @@
 		if(critter.health == critter.maxHealth)
 			to_chat(user, "<span class='notice'>[M] is at full health.</span>")
 			return
-		if(heal_brute < 1)
+		if(heal_brute < 0.1)
```

Comment:
Use the define for this (minimum damage something or the other)

Reply:
Haven't been able to find it.